css/style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+body {
```

Comment:
Please make sure to always leave an empty line after creating a CSS block to keep your code clean.